tools/generator/SourceWriters/BoundPropertyStringVariant.cs

```diff
@@ -46,9 +48,20 @@ public BoundPropertyStringVariant (Property property, CodeGenerationOptions opt)
 				SetBody.Add ($"{property.AdjustedName} = jlsa;");
 				SetBody.Add ($"foreach (var jls in jlsa) if (jls != null) jls.Dispose ();");
 			} else {
-				SetBody.Add ($"var jls = value == null ? null : new global::Java.Lang.String (value);");
-				SetBody.Add ($"{property.AdjustedName} = jls;");
-				SetBody.Add ($"if (jls != null) jls.Dispose ();");
+				IsUnsafe = true;
+				var method = property.Setter;
+				var parameter = method.Parameters [0];
```

Comment:
Is there a recommended way to assert there is 1 parameter for this property setter?

Reply:
I think it's safe to assume that if you've made it to this point in the code then the original method is valid for being a property.  (That is, it has only one parameter.)